draft-ietf-quic-ack-frequency.md

```diff
@@ -260,26 +260,11 @@ have different values in all fields. An endpoint MUST use a sequence number of 0
 for the first ACK_FREQUENCY frame it constructs and sends, and a strictly
 increasing value thereafter.
 
-An endpoint MUST allow reordered ACK_FREQUENCY frames to be received and
-processed, see {{Section 13.3 of QUIC-TRANSPORT}}.
-
-On the first received ACK_FREQUENCY frame in a connection, an endpoint MUST
-immediately record all values from the frame. The sequence number of the frame
-is recorded as the largest seen sequence number. The new Ack-Eliciting Threshold
-and Request Max Ack Delay values MUST be immediately used for delaying
-acknowledgements; see {{sending}}.
-
-On a subsequently received ACK_FREQUENCY frame, the endpoint MUST check if this
-frame is more recent than any previous ones, as follows:
-
-- If the frame's sequence number is not greater than the largest one seen so
-  far, the endpoint MUST ignore this frame.
-
-- If the frame's sequence number is greater than the largest one seen so far,
-  the endpoint MUST immediately replace old recorded state with values received
-  in this frame. The endpoint MUST start using the new values immediately for
-  delaying acknowledgements; see {{sending}}. The endpoint MUST also replace the
-  recorded sequence number.
+An endpoint might receive an ACK_FREQUENCY frame out of order 
+(see {{Section 13.3 of QUIC-TRANSPORT}}). 
+The endpoint MUST ignore the frame if the Sequence Number is smaller than the last processed Sequence Number.
+Otherwise, the endpoint MUST update its state with values received
+in this frame, including the processed largest Sequence Number.
```

Comment:
To make it even shorter we could just remove the last sentence here or use this sentence instead:
```suggestion
The endpoint MUST only process the other fields
 if the Sequence Number is larger than the last processed Sequence Number.
```
However, "MUST only" is probably not great to use for normative requirements....

Reply:
How about "MUST process the other fields if and only if the Sequence Number is larger than the last and largest processed Sequence Number."